tests/fakeapi/hello.py

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env python3
+import sys
+import unittest
```

Comment:
This imports are not being used.

Reply:
Ah, yeah, forgot to remove those from the attempt to use uniftest.skipIf.